spec/rails_helper.rb

```diff
@@ -21,7 +20,7 @@
 # directory. Alternatively, in the individual `*_spec.rb` files, manually
 # require only the support files necessary.
 #
-# Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
+Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
```

Comment:
was thinking more about newer contributors. There's overhead in doing this too if it gets big enough. I'll revert change for now since our spec size is pretty small